docs/changelog.rst

```diff
@@ -2,6 +2,9 @@
 Changelog
 =========
 
+* :release:`5.4.2 <2019-06-20>`
```

Comment:
nit: We usually release as a separate commit. There's no need to include this for now (also, technically this is a not a bugfix, so it should be a minor bump)